src/components/table/LightTableHead.tsx

```diff
@@ -40,44 +83,99 @@ function LightTableHead({
   classes,
   columns,
   onRequestSort,
+  onResizeColumn,
   orderDirection,
   orderProperty,
 }: LightTableHeadProps) {
+  const [initialWidth, setInitialWidth] = React.useState<number | undefined>(undefined);
+  const [isDragging, setDragging] = React.useState(false);
+
   const createSortHandler = (property: string) => (event: any) => {
     onRequestSort(event, property);
   };
 
+  const createStartHandler: (column: TableColumnType) => DraggableEventHandler = (column) => (
+    _,
+    data,
+  ) => {
+    if (column.width === undefined) {
+      setInitialWidth(data.node?.parentElement?.clientWidth);
+    } else {
+      setInitialWidth(column.width);
+    }
+    setDragging(true);
+  };
+
+  const createStopHandler: (column: TableColumnType) => DraggableEventHandler = (column) => (
+    event,
+    data,
+  ) => {
+    setDragging(false);
+    if (initialWidth !== undefined) {
+      onResizeColumn(event, column.name, initialWidth + data.x);
+    }
+  };
+
+  const createDragHandle = (col: TableColumnType) =>
+    col.allowResize && (
+      <Draggable
+        axis="x"
+        onStart={createStartHandler(col)}
+        onStop={createStopHandler(col)}
```

Comment:
Would you consider updating the column width while dragging is in progress?  As a user, I tend to like to resize and see the impact at the same time.

Reply:
+1, not a dealbreaker but it was something I was wondering about in your demo (thanks @otchet-broad for putting it into words 😄 )

Reply:
Yeah, I agree.  The only reason I didn't is that the Terra table doesn't do this and I've found in the past that can get kinda chunky as an interaction.  I think if I make the table virtualized, it might work but it's a pretty hefty change and wouldn't match the terra UI.  I'll mention it to Lou